bindings/matrix-sdk-crypto-js/package.json

```diff
@@ -39,6 +39,7 @@
     },
     "scripts": {
         "build": "./scripts/build.sh",
```

Comment:
Shouldn't we add `WASM_PACK_ARGS=--release` here too?

Reply:
we can do. `--release` is the default so I didn't bother.

Reply:
I prefer to have explicit behaviours rather than relying on someone knowing the default behaviours (which can change over time by the way). So please, yup, add a `WASM_PACK_ARGS=--release` if you don't mind :-). Thanks!